lib/rules.js

```diff
@@ -28,8 +37,15 @@ function createRule (data) {
 }
 
 function matchRule (rule, log) {
-  if (rule.get('conditions').every(c => matchCondition(c, log))) {
-    return rule.update('count', 0, n => n + 1)
+  if (!rule.has('error')) {
+    try {
+      if (rule.get('conditions').every(c => matchCondition(c, log))) {
+        return rule.update('count', 0, n => n + 1)
+      }
+    } catch (error) {
+      console.trace(error)
```

Comment:
At some point, we should settle on a strategy for handling and logging errors.

Reply:
Ideally you just throw errors and the pipeline take care of it. But here it is a bit different since we want to match all rules even if one errors and we also want to record if one of them error.